mlir/utils/performance/createPerformanceReports.py

```diff
@@ -18,6 +18,11 @@ def printAllPerformance(chip, lib='rocBLAS'):
             COLUMNS_TO_AVERAGE = ['MLIR TFlops', 'Tuned MLIR TFlops',
                 f'{lib} TFlops (no MLIR Kernels)', 'Tuned/Untuned',
                 f'MLIR/{lib}', f'Tuned/{lib}']
+            if 'Quick Tuned MLIR TFlops' in df:
```

Comment:
Nit, you could `+=` on the quick tuning columns instead of this redundant repeat